grammars/keywords-nb_NO.lark

```diff
@@ -0,0 +1,30 @@
+_PRINT: "skriv" | "print"
+_ASK: "spør" | "ask"
+_ECHO: "ekko" | "echo"
+_FORWARD: "forover" | "forward"
+_TURN: "snu" | "turn"
+right: "høyre" | "right"
+left: "venstre" | "left"
+_IS: "er" | "is"
+_SLEEP: "sov" | "sleep"
+_ADD_LIST: "legg til" | "add"
+_TO_LIST: "til" | "to"
+_REMOVE: "fjern" | "remove"
+_FROM: "fra" | "from"
+_AT: "på" | "at"
+random: "tilfeldig" | "random"
+_IN: "i" | "in"
+_IF: "hvis" | "if"
+_ELSE: "ellers" | "else"
+_AND: "og" | "and"
+_REPEAT: "gjenta" | "repeat"
+_TIMES: "ganger" | "times"
+_FOR: "for"
+_RANGE: "sekvens" | "range"
+_TO: "til" | "to"
+_STEP: "steg" | "step"
+_ELIF: "elhvis" | "elif"
+_INPUT: "innputt" | "input"
```

Comment:
Changed this from `innputt` to `inndata` after confeering with my co-translators 😊